batch7crud-roll8/src/main/webapp/css/layout.css

```diff
@@ -0,0 +1,101 @@
+.main-wrapper {
+  margin: 0 auto;
+  width: 100%;
+  background-color: #ECF0F5;
+  font-family: 'Source Sans Pro', sans-serif;
+  font-size: 16px;
+}
+
+/* Header */
+
+.header-wrapper {
+  background-color: #018448;
+  width: 100%;
+}
+
+.header-container {
+  padding: 10px 20px;
+}
+
+.header {
+  margin: 0 auto;
+  max-width: 1170px;
+}
+
+.header-title {
+  float: left;
+}
+
+.header-title h1 {
+  font-size: 30px;
+  font-weight: bold;
+  color: #fff;
+}
+
+.user-info {
+  float: right;
+}
+
+.user-info h2 {
+  font-size: 18px;
+  font-weight: bold;
+  color: #fff;
+}
+
+.user-menu ul li {
+  float: left;
+  padding-left: 10px;
+}
+
+.user-menu ul li:first-child {
+  border-right: solid 1px #fff;
+  padding-left: 0;
+  padding-right: 10px;
+}
+
+.user-info ul li a {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Simplify this over-specified selector [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Aoverspecific-selectors)